cli-plugins/apps/tutorials/get-workload-status.hbs.md

```diff
@@ -0,0 +1,635 @@
+>would not be included in the command output.).
+
+```bash
+# --namespace flag shorthand is -n
+# An alternative way to use this command is 
+# tanzu apps workload get tanzu-java-web-app -n development
+tanzu apps workload get tanzu-java-web-app --namespace development
+
+📡 Overview
+   name:        tanzu-java-web-app
+   type:        web
+   namespace:   development
+
+💾 Source
+   type:       git
+   url:        https://github.com/vmware-tanzu/application-accelerator-samples
+   tag:        tap-1.6.0
+   sub-path:   tanzu-java-web-app
+   revision:   tap-1.6.0/bb9e655b33e39b242b6e5d9e218578e75c1d833c
+
+
+📦 Supply Chain
+   name:   source-to-url
+
+   NAME               READY   HEALTHY   UPDATED   RESOURCE
+   source-provider    True    True      31m       gitrepositories.source.toolkit.fluxcd.io/tanzu-java-web-app
+   image-provider     True    True      30m       images.kpack.io/tanzu-java-web-app
+   config-provider    True    True      30m       podintents.conventions.carto.run/tanzu-java-web-app
+   app-config         True    True      30m       configmaps/tanzu-java-web-app
+   service-bindings   True    True      30m       configmaps/tanzu-java-web-app-with-claims
+   api-descriptors    True    True      30m       configmaps/tanzu-java-web-app-with-api-descriptors
+   config-writer      True    True      30m       runnables.carto.run/tanzu-java-web-app-config-writer
+
+🚚 Delivery
+   name:   delivery-basic
+
+   NAME              READY   HEALTHY   UPDATED   RESOURCE
+   source-provider   True    True      30m       imagerepositories.source.apps.tanzu.vmware.com/tanzu-java-web-app-delivery
+   deployer          True    True      30m       apps.kappctrl.k14s.io/tanzu-java-web-app
+
+💬 Messages
+   No messages found.
+
+🛶 Pods
+   NAME                                        READY   STATUS      RESTARTS   AGE
+   tanzu-java-web-app-build-11-build-pod       0/1     Completed   0          6d12h
+   tanzu-java-web-app-build-12-build-pod       0/1     Completed   0          22h
+   tanzu-java-web-app-build-3-build-pod        0/1     Completed   0          60d
+   tanzu-java-web-app-config-writer-655rb-pod  0/1     Completed   0          21d
+   tanzu-java-web-app-config-writer-7h8bn-pod  0/1     Completed   0          6d12h
+   tanzu-java-web-app-config-writer-7xr6m-pod  0/1     Completed   0          60d
+   tanzu-java-web-app-config-writer-g9gp8-pod  0/1     Completed   0          45d
+
+🚢 Knative Services
+   NAME                READY   URL
+   tanzu-java-web-app  Ready   http://tanzu-java-web-app.default.127.0.0.1.nip.io
+
+To see logs: "tanzu apps workload tail tanzu-java-web-app --namespace development --timestamp --since 1h"
+```
+
+## Tail logs
+
+### tanzu apps workload tail
+
+`tanzu apps workload tail` checks the runtime logs of a workload. The workload can be filtered
+by the namespace it was created on by using the `--namespace` flag, and the workload logs
+can also be filtered by a specific component. 
```

Comment:
can also be filtered by a specific `--component`.

Reply:
changed